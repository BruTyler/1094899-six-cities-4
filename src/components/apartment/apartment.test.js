import React from 'react';
import renderer from 'react-test-renderer';
import Apartment from './apartment.jsx';

const APARTMENTNAME = `apart1`;

describe(`Apartment render suit`, () => {
  it(`Apartment render apartment item case`, () => {
    const generatedTree = renderer.create(
        <Apartment
          item={APARTMENTNAME}
        />
    ).toJSON();

    expect(generatedTree).toMatchSnapshot();
  });

});
