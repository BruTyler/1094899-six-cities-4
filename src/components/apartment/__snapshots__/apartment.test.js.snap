// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Apartment render suit Apartment render apartment item case 1`] = `
<div>
  <img
    height="40"
    src="img/no-places.png"
    width="50"
  />
  <a
    className="apartment__title"
    href="#"
  >
    apart1
  </a>
</div>
`;
